src/seedu/addressbook/data/person/Phone.java

```diff
@@ -56,4 +56,10 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString(){
+        return "Phone: " +value;
```

Comment:
It would be better if it just returns the value and not the prefix. This implementation may be an issue if you're doing things like internationalisation (support for other languages) and when one needs the just the phone number in printable format.